### PR TITLE
Pin version of ltag for go <= v1.15 compatibility

### PR DESCRIPTION
**Pull request by Joffref**
Signed-off-by: Mathis Joffre <mariusjoffre@gmail.com>

*Issue #, if available:* #364

Sorry for multiple PR opening I had some troubles with my branches and squashing. This the final PR for this issue.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Kern-- — COMMENTED** [review]
Thanks for this!

If v0.2.3 still works with newer versions of go, I would prefer we just pin it for all go versions. We're already looking at removing support for <=1.15 in https://github.com/firecracker-microvm/firecracker-go-sdk/pull/440 at which point we can update the pin.

**Comment by Joffref** [comment]
I think it is the best option I will made changes, plus we should consider to pin a version for each dependency to avoid situation like this one.

File: Makefile [hunk 1]
```diff
@@ -144,6 +144,6 @@ firecracker-clean:
 
 lint: deps
 	gofmt -s -l .
-	$(FC_TEST_DATA_PATH)/ltag -check -v -t .headers
+	$(FC_TEST_DATA_PATH)/bin/ltag -check -v -t .headers
```
Comment (ginglis13) [inline Makefile:147]:
```suggestion
	$(FC_TEST_BIN_PATH)/ltag -check -v -t .headers
```

**Review by ginglis13 — APPROVED** [review]
small nit, otherwise LGTM

**Review by Kern-- — APPROVED** [review]

**Review by kzys — APPROVED** [review]

**Pull request merged**